command/command.go

```diff
@@ -23,9 +23,15 @@ var version = "v1.0.0-rc.1"
 // empty context
 var nocontext = context.Background()
 
+func new() {
```

Comment:
Consider using kingpin.Parse instead of kingpin.MustParse to handle errors instead of exiting the program.